frontend/src/pages/admin/InstanceTypes/index.vue

```diff
@@ -0,0 +1,159 @@
+<template>
```

Comment:
Assuming this file is not new and GitHub hasn't tracked the move

Reply:
Correct - I think the combination of rename *and* the number of changes of Project->Instance has exceeded what git thinks is a move and edit.